src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -26,6 +26,13 @@
                             id="host-actions"
                             [tableActions]="tableActions">
           </cd-table-actions>
+          <button *ngIf="showExpandClusterBtn" 
+                  class="btn btn-light"
+                  name="expand-cluster"
+                  aria-label="Expand Cluster"
+                  [routerLink]="'/expand-cluster'" i18n>
+                Expand Cluster
+          </button>
```

Comment:
one suggestion I want to make over here would be that we have a pattern of adding extra table actions using the `cd-table-actions` component. We do that in the [osd-list.component.html](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html#L24). Can we do the same here as well so whenever we want to add more extra actions here we don't need to add more buttons. Instead we can just add more entries to the actions list itself. WDYT?

Reply:
yes, I actually worked on that approach in beginning , but somehow my button was not getting enabled due to some permission issue or what, maybe with your help I can configure it

Reply:
if you put `[permission]="permissions.hosts"` probably that should be fine. have you tried that?

Reply:
or we also have one directive which is `cdScope`, so if you put maybe `*cdScope="'hosts'"` probably that should also work (I prefer the directive and eventually we should move all our permission stuffs to this directive.

Reply:
thanks, its working now